### PR TITLE
[ServiceBus] Bump uAMQP Dependency to v1.2.13

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

File: sdk/servicebus/tests.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: servicebus
+      InjectedPackages: $(WheelLocationsList)
```
Comment (yunhaoling) [inline sdk/servicebus/tests.yml:7]:
```suggestion
```

**Comment by yunhaoling** [comment]
python - servicebus - tests

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@
 **BugFixes**
 
 * `forward_to` and `forward_dead_lettered_messages_to` will no longer cause authorization errors when used in `ServiceBusAdministrationClient` for queues and subscriptions.
+* Updated uAMQP dependency to 1.2.13.
+  - Fixed bug in accessing message properties triggering segmentation fault when the underlying C bytes are NULL.
+  - Fixed bug where connection establishment on macOS with Clang 12 triggering unrecognized selector exception.
+  - Fixed bug that macOS was unable to detect network error.
+  - Fixed bug that ReceiveClient and ReceiveClientAsync receive messages during connection establishment.
```
Comment (annatisch) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:12]: Are there azure-sdk-python repo issue numbers we can reference here?

File: sdk/servicebus/azure-servicebus/CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,12 @@
 
 **BugFixes**
 
-* `forward_to` and `forward_dead_lettered_messages_to` will no longer cause authorization errors when used in `ServiceBusAdministrationClient` for queues and subscriptions.
+* `forward_to` and `forward_dead_lettered_messages_to` will no longer cause authorization errors when used in `ServiceBusAdministrationClient` for queues and subscriptions (#15543).
+* Updated uAMQP dependency to 1.2.13.
+  - Fixed bug that macOS was unable to detect network error (#15473).
+  - Fixed bug that ReceiveClient and ReceiveClientAsync receive messages during connection establishment (#15555).
```
Comment (yunhaoling) [inline sdk/servicebus/azure-servicebus/CHANGELOG.md:10]:
```suggestion
  - Fixed bug that `uamqp.ReceiveClient` and `uamqp.ReceiveClientAsync` receive messages during connection establishment (#15555).
```

File: sdk/servicebus/azure-servicebus/CHANGELOG.md (hunk 2 above)
Comment (yunhaoling) [reply sdk/servicebus/azure-servicebus/CHANGELOG.md]: updated, thanks for reminding.

btw, do you think it's a good idea to  mention those SB issues in the EH changelog as both SDKs depends on the same uamqp?

**Review by annatisch — APPROVED** [review]

**Review by rakshith91 — APPROVED** [review]
LGTM

**Pull request merged**